lms/djangoapps/shoppingcart/tests/test_reports.py

```diff
@@ -7,6 +7,7 @@
 import pytz
 import StringIO
 from textwrap import dedent
+from mock import patch
```

Comment:
@cahrens its not an unnecassary import it is used in patching inside the setUp()